### PR TITLE
[Sprint 1] TimeSheet UI 3

### PR DESCRIPTION
**Pull request by nguyenminhduong**
redmine https://dev.framgia.com/redmine/issues/118420
Fix ui timesheet

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/screen/demo/DemoTimeSheetActivity.java [hunk 1]
```diff
@@ -29,18 +29,44 @@ protected void onCreate(Bundle savedInstanceState) {
         timeSheetView.setTime(4, 2017);
 
         List<TimeSheetDate> timeSheetDates = new ArrayList<>();
+        timeSheetDates.add(new TimeSheetDate("2017/04/26", 8, 10, TimeSheetDate.Status.NORMAL));
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/demo/DemoTimeSheetActivity.java]: Please move to method in TimesheetRepository
We will fake these data when call from server

File: app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java [hunk 2]
```diff
@@ -232,7 +233,7 @@ protected void drawMonthNums(Canvas canvas) {
             if (mHasToday && (mToday == dayCheck)) {
                 mMonthNumPaint.setColor(mCurrentDayTextColor);
                 mMonthNumPaint.setTypeface(Typeface.create(mDayOfMonthTypeface, Typeface.NORMAL));
-                mSelectedCirclePaint.setColor(Color.parseColor("#00ced1"));
+                mSelectedCirclePaint.setColor(Color.parseColor("#96c2f2"));
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java]: I think you should put a color code in ``colors.xml``. It look better and easy for change

File: app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java [hunk 3]
```diff
@@ -249,43 +250,64 @@ protected void drawMonthNums(Canvas canvas) {
...
+                                drawDayLabelNomal(canvas, y, day, x);
+                                break;
+                            case FORGOT_CHECK_IN_CHECK_OUT_MORE_FIVE_TIME:
+                                mSelectedCirclePaint.setColor(
+                                        mDayForgotCheckInCheckOutFiveTimeColor);
+                                drawDayLabelNomal(canvas, y, day, x);
+                                break;
+                            case DAY_OFF_RO:
+                                mSelectedCirclePaint.setColor(
+                                        getResources().getColor(R.color.color_dark_gray));
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java]: Please change get colorCode from resource to from variable get from an attribute.
It easy for change later. If requirement change

File: app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java [hunk 4]
```diff
@@ -249,43 +250,64 @@ protected void drawMonthNums(Canvas canvas) {
...
+                                drawDayLabelRoP(canvas, y, day, x,
+                                        getResources().getString(R.string.day_off_ro));
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java]: like above

File: app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java [hunk 5]
```diff
@@ -249,43 +250,64 @@ protected void drawMonthNums(Canvas canvas) {
...
+                                break;
+                            case DAY_OFF_HALF_RO:
+                                mSelectedCirclePaint.setColor(
+                                        getResources().getColor(R.color.color_dark_gray));
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/widget/timesheet/TimeSheetView.java]: .

File: app/src/main/java/com/framgia/wsm/screen/demo/DemoTimeSheetActivity.java (hunk 1 above)
Comment (nguyenminhduong) [inline app/src/main/java/com/framgia/wsm/screen/demo/DemoTimeSheetActivity.java]: Cái này em chỉ test thử trong cái demo của anh thôi ạ, còn phần fake data em sẽ làm ở pull sau

Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/demo/DemoTimeSheetActivity.java]: OK

**Pull request merged**